src/test/java/ru/training/at/hw2/ex1/JdiPageSeleniumFirstTest.java

```diff
@@ -0,0 +1,194 @@
+import org.testng.asserts.SoftAssert;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.Assert.assertEquals;
+
+public class JdiPageSeleniumFirstTest {
+    WebDriver webDriver;
+    final String testPageUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+    final String userName = "Roman";
+    final String password = "Jdi1234";
+
+    @BeforeMethod(alwaysRun = true)
+    public void driverSetup() {
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Drivers\\chromedriver_win32\\chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        /*webDriver.manage().timeouts()
+                .implicitlyWait(10, TimeUnit.SECONDS);*/
+    }
+
+    @Test
+    public void jdiSeleniumPageFirstTest() {
+
+        // 1. Assert that test site page is loaded
+        webDriver.manage().timeouts().pageLoadTimeout(10, TimeUnit.SECONDS);
+        webDriver.navigate().to(testPageUrl);
+        SoftAssert softAssert = new SoftAssert();
+        WebDriverWait webDriverWait = new WebDriverWait(webDriver, 10);
+        softAssert.assertEquals(testPageUrl, webDriver.getCurrentUrl());
+
+        //2. Assert that browser title is "Home page"
+        softAssert.assertEquals(webDriver.getTitle(), "Home Page");
+
+        //3. Assert that user is logged
+        WebElement loginDropdown = webDriverWait
+                .until(webDriver -> webDriver.findElement(By.xpath("//a[@href='#']")));
+        loginDropdown.click();
+
+        WebElement username = webDriverWait
+                .until(webDriver -> webDriver.findElement(By.cssSelector(("input#name"))));
+        username.sendKeys(userName);
+
+        WebElement pass = webDriverWait
+                .until(webDriver -> webDriver.findElement(By.cssSelector(("input#password"))));
+        pass.sendKeys(password);
+
+        WebElement loginButton = webDriverWait
+                .until(webDriver -> webDriver.findElement(By.cssSelector(("button#login-button"))));
+        loginButton.click();
+
+        WebElement logout = webDriverWait
+                .until(webDriver -> webDriver.findElement(
+                        By.cssSelector(("div[class='logout'] button"))));
+        softAssert.assertTrue(logout.isEnabled());
+
+
+        //4.Assert that user name is displayed and is equal to expected
+        WebElement logged = webDriverWait
+                .until(webDriver -> webDriver.findElement(
+                        By.cssSelector(("span#user-name"))));
+        softAssert.assertTrue(logged.isDisplayed());
+        softAssert.assertEquals(logged.getText(), "ROMAN IOVLEV");
+
+        //5. Assert that there are 4 items on the header and they correspond to expected
+        List<WebElement> headerElements = webDriverWait
+                .until(webDriver -> webDriver.findElements(
+                By.xpath("//header/div/nav/ul[1]/li")));
```

Comment:
Please do not use indexes in the paths. Logic of them could be broken, if element will be added or deleted before

Reply:
changed the locator